src/thematic/1_water/sfp-105-nl_bemonsteren-peilbuis/07_stappenplan.Rmd

```diff
@@ -0,0 +1,150 @@
+# Werkwijze
+
+## Uitvoering
+
+```{=html}
+<!--
+Gedetailleerde omschrijving van alle stappen die doorlopen moeten worden om het protocol uit te voeren.
+Subtitels gebruiken om elke stap te omschrijven.
+-->
+```
+### Stappenplan
+
+De werkwijze voor het bemonsteren van een piëzometer voor een grondwatermonsters bestaat uit volgende stappen:
+
+-   Reinigen van de peilbuis of piëzometer
+
+-   Bemonsteren van een peilbuis of piëzometer
+
+-   Bepalen van de ogenblikkelijke variabelen
+
+-   Vullen van de recipiënten
+
+-   Bewaren en transporteren van watermonsters
+
+### Reinigen van een piëzometer
+
+Voor de staalname wordt de piëzometer eerst gereinigd met een borstel en daarna leeggepompt.
+
+De piëzometer wordt eerst geschuurd met een flessenborstel gemonteerd op een lange steel.
+Na het inbrengen van die flessenborstel in de buis, wordt flink op en neer bewogen over de gehele lengte van de buis, om vastzittende bacterie-kolonies en andere contaminanten van de buis los te schuren en eventueel gesedimenteerd materiaal terug in suspensie te brengen.
+
+Daarna wordt (in de mate van het mogelijke) al het water met een peristaltische pomp uit de piëzometer gepompt.
+De telfondarm wordt volledig tot op de bodem van de peilbuis gebracht.
+Zo wordt eventueel slib uit de piëzometer afgevoerd.
+Het pompen zorgt ervoor dat vers grondwater wordt bemonsterd.
+Bij beperkte watertoevoer (wat bij deze piëzometers doorgaans het geval is) naar de buis zal de piëzometer na enkele minuten leeggepompt zijn.
+Het opgepompte water wordt minstens twee meter van de locatie van de peilbuis afgevoerd.
+
+????Iets over de snelheid van leegpompen toe te voegen Ja????
+
+### Bemonsteren van een peilbuis of piëzometer
+
+Het bemonsteren van een peilbuis gebeurt in een periode van 1 tot 3 dagen na het reinigen van de peilbuis.
+De bemonstering gebeurt met een peristaltische pomp of kogelklepbemonsteraar.
+
+Bij het nemen van een monster met een peristaltische pomp, wordt een teflondarm tot op een diepte van 10 cm onder het waterpeil van de peilbuis gebracht.
+De teflondarm wordt verbonden met de peristaltische pomp en het 'vers' grondwater wordt met een matige snelheid opgepompt zodat geen gasbellen in het opgepompte water terechtkomen.
+Er moet absoluut vermeden worden dat de peilbuis leeggepompt wordt tijdens de staalname.
+Het opgepompte water wordt opgevangen in een recipiënt (monsternamepot) totdat een volume van minstens 500 ml is bereikt.
+Bij voldoende wateraanvoer in de peilbuis wordt de eerste 100 ml niet opgevangen.
```

Comment:
Is parametrisatie van deze cijfers zinvol / nodig? Je kan aangeven dat deze cijfers de INBO defaultwaarden zijn, maar dat daar in projectspecifieke context van kan afgeweken worden als dat nodig is.

Reply:
Ik denk niet dat hier ooit van afgeweken wordt. Ik neem het mee in het lijstje te bediscusiëren punten